### ISSUE TITLE
Can't access plugin settings

### ISSUE THREAD

**Issue by jltnol** [IndigoDomotics/Beacon#2]
I'm moving Indigo to a different computer, so have cloned the OS Drive to a portable drive and have cloned the copy to the internal drive of the target iMac.  OS for Indigo is 10.15.6, and Indigo V7.5

I know enough about networking so have set up the target iMac to get the same IP address from the router as the previous Mini running Indigo. I'm using Geofency , on port 6192, and that port is forward on the router to the same old IP address used on the mini, now being used in the iMac running Indigo.  I've restarted everything, and access from the iOS Touch app as well as indigo web pages work just fine. Connections to those ports work as expected.

In reading up on this, it seems I should be able to have a "settings" menu in the Beacon plugin to change the port number, but don't. I've tried re-installing the plugin, but that hasn't worked either. 

From my phone, when trying to send  Enter/Exit data, all I get is a error message:  "Failed. Cannot connect to the server. "

While I guess its possible my ISP has started blocking this port, the chances of that happening at the same time as moving my Indigo server seems remote at best. 

I've also tried creating a new "fence", but even that gives the same error message.

I'm guessing since I can't access the "settings" in the plugin menu, that's where my problem is.

**Comment by FlyingDiver** [comment]
Run the Indigo installer again to make sure all the permissions are set correctly.

**Comment by jltnol** [comment]
Hey Joe

Thank you so much for your speedy reply.

I was having some other issues, so have since done what you’ve suggested… re-installed Indigo… and after that install, I re-installed the plugin for good measure as well.

Still only have Disable, Reload, Show… About… options in the plugin.

I manually deleted the plugin, re-started the computer, re-installed the plugin, but still not working. 

As far as permissions go, which ones (file/folder) might be incorrect?

FYI I’m on 1.5.3 for the plugin.

I don’t think its a networking issue… the firewall is OFF in the iMac… but just can’t figure out what be causing this to not work……. I even tried creating a new database just to see… but still not working...

Jeffrey

**Comment by FlyingDiver** [comment]
Check inside the plugin package and make sure the PluginConfig.xml file is there.  And has the right permissions.

```
jkeenan@Kraken Downloads % ls -lR Beacon.indigoPlugin
total 0
drwxr-xr-x@ 5 jkeenan  staff  160 Dec  8  2017 Contents

Beacon.indigoPlugin/Contents:
total 8
-rwxr-xr-x@ 1 jkeenan  staff  710 Dec  8  2017 Info.plist
drwxr-xr-x@ 3 jkeenan  staff   96 Dec  8  2017 Resources
drwxr-xr-x@ 6 jkeenan  staff  192 Dec  8  2017 Server plugin

Beacon.indigoPlugin/Contents/Resources:
total 40
-rw-r--r--@ 1 jkeenan  staff  17556 Dec  8  2017 icon.png

Beacon.indigoPlugin/Contents/Server plugin:
total 72
-rwxr-xr-x@ 1 jkeenan  staff   1538 Dec  8  2017 Devices.xml
-rwxr-xr-x@ 1 jkeenan  staff   1801 Dec  8  2017 Events.xml
-rwxr-xr-x@ 1 jkeenan  staff   4925 Dec  8  2017 PluginConfig.xml
-rwxr-xr-x@ 1 jkeenan  staff  16949 Dec  8  2017 plugin.py
jkeenan@Kraken Downloads % 
```

**Comment by jltnol** [comment]
Hey Joe

First of all, I’m back in business!

The short story is while doing all this migrating, and running into problems(Beacon was the one that stood out, there may have been others….), I decided to give Big Sur a try since it was already loaded on the target iMac. And since I had the Indigo 7.5 client loaded, it was simple enough to go ahead and manually move all the plugins, and their settings to Big Sur, and start the Indigo Server from there.

As of now, all is well, 

I did go back and check the permissions from the old Catalina/Indigo server, and I’m attaching a pic as the all look correct… at least they all look like my working copy on Big Sur. Still not sure what was wrong… but there could have been some OS migration issues, so I’m going back to take a look at those as well. If I find anything, I’ll keep you posted.. I honestly thought this was going to be a 20 minute task, but its taken all afternoon. But, there are worse ways to spend my time for sure. 

Again, thank you so much for developing this plug in… I use Geofency day in and day out… and I also appreciate your taking time to help with this particular issue.

Ill let you know if I find anything else.

Cheers

 Jeffrey Talbot

Here’s the link for the pic:

http://jeffreytalbot.com/indigo/Screen%20Shot%202021-04-06%20at%206.14.44%20PM.png <http://jeffreytalbot.com/indigo/Screen%20Shot%202021-04-06%20at%206.14.44%20PM.png>
